infra/resources/_modules/cdn/cdn.tf

```diff
@@ -0,0 +1,29 @@
+module "wallet_cdn" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cdn?ref=v8.12.2"
+
+  name                  = "wallet-cdn-01"
```

Comment:
Shouldn't be `io-p-wallet-cdn-01`?

Reply:
the module has the `prefix` attribute that does the trick